fontforge/splinestroke.c

```diff
@@ -69,6 +69,11 @@ static inline int BPNear(BasePoint bp1, BasePoint bp2) {
     return BPWithin(bp1, bp2, INTRASPLINE_MARGIN);
 }
 
+// Not called "BPNear" because this is specific to UTanVecs
+static inline int UTNear(BasePoint bp1, BasePoint bp2) {
+    return BPWithin(bp1, bp2, UTMARGIN);
```

Comment:
Seems like BPWithin is only used in splinestroke.c and utanvec.c, why not move BPWithin to utanvec.h, which apparently has the rest of the utility functions on BasePoints already? Then you can move UTNear into that header too.

Reply:
This would work in the way I would prefer it to if you'll spot me a separate extern declaration of RealWithin.

Reply:
Seems fine to me to include splineutil2.h from utanvec.h, the former is just a bunch of function declarations which seems pretty light, and utanvec.h is already including splinefont.h

Reply:
OK, I'll take a look.